pkg/util/timeconv/duration/duration.go

```diff
@@ -0,0 +1,161 @@
+package duration
+
+import (
+	"strconv"
+	"time"
+)
+
+type DurationUnit string
+
+const (
+	UnitMonth         = DurationUnit("mo")
+	UnitMillisecond   = DurationUnit("ms")
+	UnitMicrosecond   = DurationUnit("us")
+	UnitMicrosecondB5 = DurationUnit("µs")
+	UnitMicrosecondBC = DurationUnit("μs")
+	UnitNanosecond    = DurationUnit("ns")
+	UnitYear          = DurationUnit('y')
```

Comment:
for clarity, make sure to use double quotes (string) rather than single (char) on all of these Unit consts.